server/kepler_model_trainer.py

```diff
@@ -22,12 +23,26 @@ def generate_regression_model(train_features) -> Sequential:
     new_linear_model.compile(optimizer=tf.optimizers.Adam(learning_rate=0.1), loss='mean_absolute_error')
     return new_linear_model
 
+# Helper function to verify model_type is valid, check whether the model has been created or not,
+# and create a filepath to the model.
+# Returns type (str, Bool) where str is the filepath to the model_type or None if the model_type
+# is invalid, and Bool is True if the model of the desired type was already created and False if the model
+# of the desired type has not been created. If str is None, then Bool is False.
+def return_model_filepath(model_type):
+    if model_type in model_type_to_filepath:
+        filepath = os.path.join(os.path.dirname(__file__), model_type_to_filepath[model_type])
```

Comment:
can the code the returns the model path be consolidated? See also [L82](https://github.com/sustainable-computing-io/kepler-model-server/pull/8/files#diff-fe87d5b32ed8308e36f1c74dbbc7e05e675778ecdcc3a1984e0ec3cc1abd201dR82)